_tuning-your-cluster/availability-and-recovery/remote-store/remote-cluster-state.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: default
+title: Remote Cluster State
+nav_order: 5
+parent: Remote-backed storage
+grand_parent: Availability and recovery
+---
+
+# Remote Cluster State
+
+Introduced 2.10
+{: .label .label-purple }
+
+Remote Cluster State protects against any cluster state metadata loss resulting due to quorum loss (permanently losing majority of cluster manager nodes) in the cluster.
+
+Cluster State in an internal data structure which contains the metadata of the cluster along with other information. 
```

Comment:
`Cluster State in an internal data structure`  -> `Cluster State **is** an internal data structure`